### PR TITLE
Fix tokenization benchmark issue

### PR DESCRIPTION
**Pull request by heytitle**
Related to #268

### CODE REVIEW

File: bin/word-tokenization-benchmark [hunk 1]
```diff
@@ -21,7 +21,7 @@ parser.add_argument(
 parser.add_argument(
     "--test-file",
     action="store",
-    help="path to test file"
+    help="path to test file, i.e. ground truth"
```
Comment (wannaphongcom) [inline bin/word-tokenization-benchmark]: Can you add type file?

Comment (heytitle) [reply bin/word-tokenization-benchmark]: Both input and test files are plain text files.

File: bin/word-tokenization-benchmark [hunk 2]
```diff
@@ -15,13 +15,13 @@ parser = argparse.ArgumentParser(
...
-    help="path to file that you want to compare against the test file"
+    help="path to file (.txt) that you want to compare against the test file"
 )
 
 parser.add_argument(
     "--test-file",
     action="store",
-    help="path to test file, i.e. ground truth"
+    help="path to test file (.txt), i.e. ground truth"
 )
```
Comment (heytitle) [inline bin/word-tokenization-benchmark]: @wannaphongcom  do you think it's ok to have the file type like this?

**Comment by bact** [comment]
On Windows, there is Unicode encoding problem at line 11 of `tests/test_benchmarks.py`.
May need to change open mode to `"rb"`

```python 
with open("./tests/data/sentences.yml", "rb") as stream:
    TEST_DATA = yaml.safe_load(stream)
```

**Comment by bact** [comment]
`load_dict()` in `attacut\utils.py` (line 92) may need to specify encoding (utf-8) when open file.

**Comment by heytitle** [comment]
@bact I've updated the attacut package with the load_dict patch. At the moment, it's a pre-release version. 

https://pypi.org/project/attacut/1.0.2.dev0/

**Comment by heytitle** [comment]
@bact I've renamed the benchmark module to `word_tokenization`. 
Please note here that I didn't change this part: https://github.com/PyThaiNLP/pythainlp/pull/269/files#diff-4a1ef3fd7db1693d63243fd8a5ecb972R214, because it's an attribute that's used on [the visualisation web][web].

[web]: https://pythainlp.github.io/tokenization-benchmark-visualization/

**Comment by bact** [comment]
Thanks!

File: docs/api/benchmarks.rst [hunk 3]
```diff
@@ -19,6 +19,6 @@ Quality
 
    Qualitative evaluation of word tokenization.
 
-.. autofunction:: pythainlp.benchmarks.word_tokenisation.compute_stats
-.. autofunction:: pythainlp.benchmarks.word_tokenisation.benchmark
-.. autofunction:: pythainlp.benchmarks.word_tokenisation.preprocessing
+.. autofunction:: pythainlp.benchmarks.word_tokenization.compute_stats
+.. autofunction:: pythainlp.benchmarks.word_tokenization.benchmark
+.. autofunction:: pythainlp.benchmarks.word_tokenization.preprocessing
```
Comment (heytitle) [inline docs/api/benchmarks.rst]: oh, thanks!

**Pull request merged**